src/raftstore/store/config.rs

```diff
@@ -17,8 +17,8 @@ use std::time::Duration;
 use raftstore::Result;
 
 const RAFT_BASE_TICK_INTERVAL: u64 = 100;
-const RAFT_HEARTBEAT_TICKS: usize = 3;
-const RAFT_ELECTION_TIMEOUT_TICKS: usize = 15;
+const RAFT_HEARTBEAT_TICKS: usize = 10;
+const RAFT_ELECTION_TIMEOUT_TICKS: usize = 50;
```

Comment:
I think we should leave this to user. This is hard code in the moment, and will be configurable later.